### ISSUE TITLE
Question regarding recoverPose

### ISSUE THREAD

**Issue by thomasstats** [magicleap/SuperGluePretrainedNetwork#19]
In the code it has a focal length manually input as 1e9.  I just wanted to see if there's any clarification or justification for this, as typically this is default to 1.  I couldn't find anything explicitly stated in the paper regarding this.

Edit:  I guess another question add on top of this is why is the bias set to True on the MLP layers if it is by default using BatchNorm?

**Comment by Skydes** [comment]
1. The 5th argument of `cv2.recoverPose` is a [distance threshold](https://docs.opencv.org/3.4/d9/d0c/group__calib3d.html#ga2ee9f187170acece29c5172c2175e7ae) to exclude points that are too far away in the inlier count. We set it to a high value so that this filtering is disabled.

2. You are right - the bias of the convolution is not necessary if batch norm is used right afterwards. That is an oversight, but it doesn't have any impact on the performance.

**Comment by thomasstats** [comment]
@Skydes thanks for the clarification.

**Comment by Skydes** [comment]
You're welcome. Please close the issue if it is solved.
